src/Microsoft.OData.Core/Json/ODataJsonWriterUtilsAsync.cs

```diff
@@ -0,0 +1,274 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataJsonWriterUtilsAsync.cs" company="Microsoft">

```

Comment:
>ODataJsonWriterUtilsAsync.cs [](start = 20, length = 28)

since your file name is 'ODataJsonWriterUtilsAsync.cs", might be use the same name for the class name?

Reply:
This is the async part of the partial ODataJsonWriterUtils class -- not a separate class.  Alternatively, we could just add these methods to ODataJsonWriterUtils.cs.

---
In reply to: [600024664](https://github.com/OData/odata.net/pull/2031#discussion_r600024664) [](ancestors = 600024664)